accelerator/roles/intel/tasks/install_ubuntu.yml

```diff
@@ -47,6 +47,15 @@
         name: "{{ intel_habana_packages | list }}"
         update_cache: true
 
+    - name: Remove (old) Gaudi kernel modules if present
+      community.general.modprobe:
+        name: "{{ item }}"
+        state: absent
+      loop: "{{ intel_gaudi_kernel_module_to_load }}"
+
+    - name: Make sure dkms module is built for the current kernel
+      command: /usr/lib/dkms/dkms_autoinstaller start
```

Comment:
python3.11 -m pip install ansible-lint==6.22.0

Please run ansible-lint for the task

Reply:
Done